### ISSUE TITLE
Menu has an odd interim stage on resizing

### ISSUE THREAD

**Issue by icklecows** [eprintsug/bootstrap#17]
The main navigation has an interim stage between the mobile three-line menu and the full size sidebar.  It puts the whole thing full-width inbetween those two states, which takes up most of the screen.
![menu-interim](https://cloud.githubusercontent.com/assets/7404917/8827904/42a191f0-3087-11e5-8c34-03ab0e6a8844.png)

**Comment by davidmce** [comment]
Thats not great looking.. 
I suspect it might be somethjing to do with the way I've had to manipulate the menus. Basically, to have a mobile menu, jquery copies the menu to a new location on the page, then deletes the original menu. 

Its possible that your screen resolution isn't one i tested.

**Comment by denics** [comment]
Hi @davidmce , if you are interested I also rewrote the menu routine to generate a menu with bootstrap tags (no more need for menu.js) and render something close to drupal admin_menu , a top black bar. If you want I can do a pull request.

**Comment by davidmce** [comment]
@denics sounds interesting! can we see a screenshot?

**Comment by denics** [comment]
Ok, perhaps the red buttons are too much :) but you can customise them!

![screen shot 2015-08-05 at 15 57 17](https://cloud.githubusercontent.com/assets/2407163/9087444/f3a6db46-3b8a-11e5-829c-11d4a1696910.png)

**Comment by davidmce** [comment]
This might make more sense than the way it is at present. Would be good to have it match the installed bootstrap theme though

**Comment by denics** [comment]
yep, we can clean the code before posting. I will be working on this asap
